src/components/mainLanding/mainNav.tsx

```diff
@@ -39,6 +41,10 @@ export default function MainLandingNav({ user, logOut }: HeaderProps) {
     return () => window.removeEventListener('scroll', handleScroll)
   }, [])
 
+  if (pathname.startsWith('/admin') || pathname.startsWith('/auth')) {
```

Comment:
why are we hiding this here?

this isnt the way... page.tsx decides where or when we show this based on the route.